pkg/util/system/cgroup_test.go

```diff
@@ -83,46 +83,6 @@ func TestCgroupFileReadInt(t *testing.T) {
 
 }
 
-func genCPUStatContent() string {
-	return "nr_periods 18491717\n" +
-		"nr_throttled 12\n" +
-		"throttled_time 123\n"
-}
-
-func TestGetCPUStatRaw(t *testing.T) {
```

Comment:
`GetCPUStatRaw` is still in use for retrieving `cfs throttled`. The file `cpu.stat` is different from `cpuacct.stat`.

Reply:
OK, but `make lint` give me this:
```
pkg/util/system/cgroup_test.go:86:6: `genCPUStatContent` is unused (deadcode)
func genCPUStatContent() string {
     ^
```

Reply:
It's a typo, it should read `CPUStat` here. fixed.

Reply:
ok